sdk/cdn/Azure.ResourceManager.Cdn/src/Generated/Models/WafPolicySettings.Serialization.cs

```diff
@@ -55,15 +55,22 @@ void IJsonModel<WafPolicySettings>.Write(Utf8JsonWriter writer, ModelReaderWrite
             }
             if (Optional.IsDefined(DefaultCustomBlockResponseBody))
             {
-                writer.WritePropertyName("defaultCustomBlockResponseBody"u8);
+                if (DefaultCustomBlockResponseBody != null)
```

Comment:
Modeler4 emit `DefaultCustomBlockResponseBody` is nullable. so this change is expected